celery/readme.txt

```diff
@@ -1 +0,0 @@
-This directory is used for celery beat scheduler to write celerybeat-schedule file for scheduling purposes.
```

Comment:
Please don't remove this file but keep this readme.txt file there for informational purpose.